src/components/toolbar/toolbar.scss

```diff
@@ -0,0 +1,66 @@
+@import "variables";
+@import "mixins";
+@import "default-theme"; //TODO: remove that soon.
+
+$toolbar-min-height: 64px !default;
+$toolbar-font-size: 20px !default;
+$toolbar-padding: 16px !default;
+
+:host {
+  display: flex;
+  box-sizing: border-box;
+
+  width: 100%;
+  min-height: $toolbar-min-height;
+
+  // Font Styling
+  font-size: $toolbar-font-size;
+  font-weight: 400;
+  font-family: $md-font-family;
+
+  padding: 0 $toolbar-padding;
+
+  // Flexbox Vertical Alignment
+  flex-direction: row;
+  align-items: center;
+
+  background: md-color($md-background, app-bar);
+  color: md-color($md-foreground, text);
+
+  &.md-primary {
+    background: md-color($md-primary);
+    color: md-color($md-primary, default-contrast);
+  }
+
+  &.md-accent {
+    background: md-color($md-accent);
+    color: md-color($md-accent, default-contrast);
+  }
+
+  &.md-warn {
+    background: md-color($md-warn);
+    color: md-color($md-warn, default-contrast);
+  }
+
+  // This allows the user to dynamically apply different rows for the toolbar.
+  &.md-toolbar-rows {
```

Comment:
I think this could be a little more intelligent. Something like...

```html
<!-- basic toolbar -->
<md-toolbar>
  <!-- buttons and labels -->
</md-toolbar>

<!-- multiple rows -->
<md-toolbar>
  <!-- stuff on the first row -->
  <md-toolbar-row>
    <!-- stuff on the second row -->
  </md-toolbar-row>
</md-toolbar>
```

@hansl @robertmesserle @kara what do you think?